docs/concepts/feathr-concepts-for-beginners.md

```diff
@@ -126,9 +126,18 @@ client.get_online_features(feature_table = "agg_features",
 ## Illustration
 
 An illustration of the concepts and process that we talked about is like this:
-![Feature Join Process](../images/observation_data.jpg)
+![Observation Data and Feature Query Process](../images/observation_data.jpg)
 
-## Point in time joins and aggregations
+## Miscellaneous topics
```

Comment:
`Miscellaneous` looks confusing to me as a `beginner concept` document section header.
Will something like `FAQ` more suitable?